streaming/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@mastodon/streaming",
+  "license": "AGPL-3.0-or-later",
+  "packageManager": "yarn@4.0.1",
+  "engines": {
+    "node": ">=18"
+  },
+  "description": "Mastodon's Streaming Server",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mastodon/mastodon.git"
+  },
+  "scripts": {
+    "start": "node ./index.js"
```

Comment:
In the future we might want to have more script here, to build / typecheck / lint / … this part of the project independently. But at the moment, the root commands already cover `streaming/`, and I am not sure how to set it up properly to have them separate yet.

Reply:
yeah, so, the root command could be now changed to `yarn workspace @mastodon/streaming start` instead of `cd streaming && yarn start`

Reply:
But we should also consider `yarn start` at the root largely deprecated in favor of `yarn workspace @mastodon/streaming start`, but we need the shim in there for people who haven't migrated.